### ISSUE TITLE
text size too small

### ISSUE THREAD

**Issue by mrba59** [faustomorales/keras-ocr#70]
Hi faustomorales, 
Firstly i want to thank you for your work.
I am dealing with the case where the text size on the image is small.
I have this configuration for image reading:

```tools=keras_ocr.tools
images=[image0,image1,image2,image3,image4, image5, image6]
images = [tools.read(image) for image in images]
images = [tools.resize_image(image, max_scale=2, max_size=4096) for image in images]
max_height, max_width = np.array([image.shape[:2] for image, scale in images]).max(axis=0)
scales = [scale for _, scale in images]
images = np.array([tools.pad(image, width=max_width, height=max_height) for image, _ in images])
boxes = detector.detect(images=images)```

The detection works perfectly but not the recognition. For this image:
![907](https://user-images.githubusercontent.com/32193116/80953461-de0e2c80-8dfb-11ea-972a-022772ac98e9.jpg)

I've got this result:

```
[['reaisateurs', 'jcrome', 'na', 'arro', 'sebastenperroy']]
[['ann', 'cimosnino']]
[['scnario', 'l', 'saptanon']]
[['lise', 'barenbaun', 'sarah', 'nrle', 'nassin', 'sen', 'allal', 's', '', 'nic', 's', 'ons', 'eyle', 'nawl', 'y', 'wo', 'dis', 'e', 's', 'o', 's', 'ogotte', 'ge', 'ric']]
[['ueer']]
[['eric', 'oueho', 'rstrllr', 'sayada']]
[['dialogurs', 'sassia']]
[['a', 's']]
[['rmr', 'duntanet', 'choe', 'ola', 'chant']]
[['rche', 'narrea', 'tive']]
[['n', '', 's', 'wlee', 'rod', 'wroimont']]
[['nicolas', 'brossette', 'direction', 'des', 'arches', 'nicolas', 'nrosscite']]
[['direction', 'dateles', 'saran', 'bellassen']]
[['dircction', 'drs', 'dialoouts', 'isabelle', 'dubernet', 'dinocuist', 'de', 'platau', 'christoehe', 'pages']]
[['producteus']]
[['sae', 'ope', 'ra', 's', 'saran', 'e', 'araand', 'une', 'prouton', 'naee', 'par']]
[['hudert', 'sesson']]
[['charoe', 'de', 'production', 'fanmicr', 'drlontite']]
[['planneun', 'francois', 'chaillou']]
[['administratrices', 'de', 'production']]
[['rie', 's', 'jot']]
[['nelanie', 'jobert', 'rtgisseur', 'central', 'frederic', 'roca']]
[['oirrcton', 'ogcasing', 'eooy', 'pasouere', 'ul', 'l', 'guy', 'ote']]
[['gilles', 'nartinez']]
[['cha', 'serc', 'ceos', 'cobot', 'ult']]
[['currs', 'optaatrurs', 'herve', 'lode', 'rodolee', 'sle', 'stan', 'e', 'spi', 'noenicurs', 'dy', 'son']]
[['dancois', 'sende']]
[['ortooirt', 'gern', 'an']]
[['jurre']]
[['cher', 'costueers', 'corinne', 'calanoee']]
[['dinecteur', 'arunstioue', 'post', 'production', 'sensamn', 'finet']]
[['responsanles', 'po', 'produon']]
[['rozenn', 'k', 'fn', 'iape', 'e', 'vee', 'vre']]
[['direction', 'de', 'l', 'cotn', 'saon', 'isabelle', 'regourd']]
```
I tried with max_scale=3 the result was different but not better , i firstly thought it was caused by the text size, but could it be caused by the color of the background ? 
In both case how can i improve the recognition accuracy  ?
Thanks

**Comment by faustomorales** [comment]
Hi there -- your code snippet appears to be missing the part where you run the recognizer. To make it easier for others to reproduce, could you share the *full* code snippet for generating the inferences you show in your output (including loading the detector, etc.)? Thanks!

**Comment by mrba59** [comment]
yeah no problem thank you for replying fast, here the code , just note that my scripts have a box processing, that clusterize the box by their positions. But in this comment i didn't put the processing part, so the recognition result is different from my first post  but, the problem is still the same.

![907](https://user-images.githubusercontent.com/32193116/80991998-2bf55580-8e39-11ea-92d7-499895822fdf.jpg)


```
detector = keras_ocr.detection.Detector(weights='clovaai_general')
recognizer = keras_ocr.recognition.Recognizer(weights='kurapan')

tools=keras_ocr.tools
image_path=' '  # put the image path
images=[image_path] 
images = [tools.read(image) for image in images]
images = [
            tools.resize_image(image, max_scale=2, max_size=4096)
            for image in images
        ]
max_height, max_width = np.array([image.shape[:2] for image, scale in images]).max(axis=0)
scales = [scale for _, scale in images]
images = np.array(
            [tools.pad(image, width=max_width, height=max_height) for image, _ in images])
boxes = detector.detect(images=images)

#plot the box 

image_r = keras_ocr.tools.read(images[0])
new_boxes = keras_ocr.tools.drawBoxes(image=image_r, boxes=boxes[0])
plt.figure()
plt.imshow(new_boxes)

sorted_rec = recognizer.recognize_from_boxes([images[0]],[boxes[0]])
print(sorted_rec)
```


here the result

```[['direction', 'dateled', 'reaisateurs', 'orection', 'orcasling', 'jcrome', 'saran', 'belnassen', 'pasouere', 'na', 'yaree', 'peooy', 'ul', 'd', 'ecn', 'guy', 'otte', 'sebasenderroy', 'dilocurs', 'dinection', 'ots', 'ginles', 'nantnez', 'clnosnine', 'ane', 'isabelle', 'dubernet', 'ghed', 'occond', 'bcue', 'sctnario', 'r', 'adapiaton', 'nanc', 'thnesault', 'dialoouiste', 'plattan', 'de', 'lise', 'barenbaun', 'christoehe', 'pages', 'sarae', 'nelo', 'ssin', 'currs', 'oprratrurs', 'nassin', 'sen', 'allal', 'herve', 'lode', 'producteus', 'stedhe', 'sanson', 'olamtd', 'cnyerilee', 'c', 'c', 'wncent', 'meslet', 'straphing', 'ooolene', 'tentl', 'c', 'oubes', 'sophie', 'ferrario', 'chens', 'd', 'hn', 'ane', 'a', 'sadah', 'arahhand', 'f', 'noenieurs', 'du', 'son', 'dibn', 'ougeoite', 'na', 'wel', 'a', 'dancois', 'sende', 'rric', 'uhere', 'iniiee', 'unt', 'proouton', 'pae', 'oatgoine', 'gern', 'ailn', 'eric', 'oueho', 'hudert', 'aesson', 'julen', 'anaud', 'rstrllr', 'sayada', 'cuaroe', 'dc', 'production', 'chef', 'costueers', 'dialogurs', 'calanort', 'drlontite', 'corinng', 'adeicr', 'edisch', 'sasiie', 'a', 'orecteur', 'armistioue', 'post', 'production', 'dunrenet', 's', 'deler', 'planneur', 'benjanin', 'hnet', 'chloe', 'chant', 'francois', 'chaillou', 'gla', 'responsanles', 'production', 'post', 'rche', 'harea', 'tive', 'administrairices', 'de', 'production', 'rozenn', 'le', 'diore', 'pape', 'wroinont', 'arie', 'jot', 'ol', 'lia', 'n', 'wllen', 'rod', 'sa', 'julia', 'avero', 'f', 'nelanie', 'jobery', 'nicolas', 'brossette', 'direction', 'or', 'la', 'cotn', 'rtoisscur', 'geneal', 'direction', 'des', 'arches', 'svaldi', 'freoeric', 'roca', 'nrosscite', 'isabelle', 'regourd', 'nicolas']]```

​
​
